test/NonSiloTests/Orleans.NonSiloTests.csproj

```diff
@@ -48,21 +48,43 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="AssemblyLoaderTests.cs" />
+    <Compile Include="AsynchAgentRestartTest.cs" />
+    <Compile Include="AsyncPipelineTests.cs" />
+    <Compile Include="Async_AsyncExecutorWithRetriesTests.cs" />
+    <Compile Include="Async_TimingTests.cs" />
+    <Compile Include="CodeGeneratorTests.cs" />
     <Compile Include="ConfigTests.cs" />
+    <Compile Include="CounterTests.cs" />
+    <Compile Include="ErrorInjectionStorageProvider.cs" />
+    <Compile Include="General\CounterStatisticTests.cs" />
+    <Compile Include="General\Identifiertests.cs" />
+    <Compile Include="General\InterfaceRules.cs" />
+    <Compile Include="General\InternalSerializationTests.cs" />
+    <Compile Include="General\LruTest.cs" />
+    <Compile Include="General\RequestContextTestsNonSiloRequired.cs" />
+    <Compile Include="General\RingTests_Standalone.cs" />
+    <Compile Include="ManagementAgentTests.cs" />
+    <Compile Include="OrleansRuntime\AsyncSerialExecutorTests.cs" />
+    <Compile Include="OrleansRuntime\ExceptionsTests.cs" />
+    <Compile Include="OrleansRuntime\Streams\BestFitBalancerTests.cs" />
+    <Compile Include="OrleansRuntime\Streams\SubscriptionMarkerTests.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="..\..\src\Build\GlobalAssemblyInfo.cs">
       <Link>Properties\GlobalAssemblyInfo.cs</Link>
     </Compile>
     <Compile Include="SchedulerTests\OrleansTaskSchedulerAdvancedTests.cs" />
     <Compile Include="SchedulerTests\OrleansTaskSchedulerAdvancedTests_Set2.cs" />
     <Compile Include="SchedulerTests\OrleansTaskSchedulerBasicTests.cs" />
-    <Compile Include="SerializationTests\BuiltInSerializerTests.cs" />
+    <Compile Include="SerializationTests\BondSerializationTests.cs" />
```

Comment:
>BondSerializationTests [](start = 41, length = 22)

hmmm, I'm a little sad we are introducing this to the new NonSiloTests project instead of in its own project, but I want to get this merged soon, so I guess it's as easy to move from here than from Tester. In upcoming PRs, please don't bring in new tests for extensions (ie: Bond is not core to Orleans, just an extension) into the new test projects that we'll start creating, as this would just split the currently 2 huge conglomerate of tests into 3, but not really vertical separation.